deflate_rle.c

```diff
@@ -34,21 +33,13 @@ Z_INTERNAL block_state deflate_rle(deflate_state *s, int flush) {
         }
 
         /* See how many times the previous byte repeats */
-        if (s->lookahead >= STD_MIN_MATCH && s->strstart > 0) {
+        if (s->lookahead >= WANT_MIN_MATCH && s->strstart > 0) {
```

Comment:
Do we actually need to change the minimum lookahead from 3 to 4 bytes here, why?
This doesn't use the 32bit hashing, so I don't quite see what the reason is for this change..?

Or is it so we have enough buffer space for running compare256_rle_unaligned32? If so, what then about the 64bit version?

Reply:
It does not need to be updated. I think at the time I was simply copying some change in the other deflate sources. I have changed it back to use `STD_MIN_MATCH`.